package.json

```diff
@@ -35,7 +35,7 @@
     "lint:css": "stylelint --ignore-path .gitignore \"{packages/**/*.{css,js},website/**/*.css}\"",
     "lint:js": "eslint --color --ignore-path .gitignore \"{{packages,scripts,website}/**/,}*.js\"",
     "lint:format": "prettier \"{{packages,scripts,website}/**/,}*.{js,css}\" --list-different",
-    "format": "run-s \"lint:css -- --fix --quiet\" \"lint:js -- --fix --quiet\" \"format:prettier -- --write\"",
```

Comment:
Stylelint chokes on nested template literals, dropping autofix of css for now (will still be linted).